msal/tests/Test.MSAL.NET.UIAutomation/MSALMobileTestHelper.cs

```diff
@@ -72,6 +72,37 @@ public void AcquireTokenSilentTestHelper(ITestController controller, LabResponse
             CoreMobileTestHelper.VerifyResult(controller);
         }
 
+        /// <summary>
+        /// Runs through the B2C acquire token flow
+        /// </summary>
+        public void B2CLocalAccountAcquireTokenInteractiveTestHelper(ITestController controller, LabResponse labResponse, string promptBehavior)
+        {
+            PrepareForAuthentication(controller);
+            SetB2CInputData(controller);
+
+            CoreMobileTestHelper.PerformB2CLocalAccountSignInFlow(controller, labResponse.User);
+            CoreMobileTestHelper.VerifyResult(controller);
+        }
+
+        /// <summary>
+        /// Runs through the standard acquire token silent flow
```

Comment:
Should update comment to reflect the usage of B2C

Reply:
done.

---
In reply to: [233218658](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r233218658) [](ancestors = 233218658)